testdata/ignore_errors/Taskfile.yml

```diff
@@ -0,0 +1,15 @@
+CmdShouldPass:
```

Comment:
This file is out of convention of other fixtures. Task names should be lowercase, and we're missing space between tasks

Also, wrap them in a `tasks:` object and add `version: '2'` at the top

Reply:
No problem